### ISSUE TITLE
Importing CSV with less columns than table causes exception

### ISSUE THREAD

**Issue by negora** [dbeaver/dbeaver#7963]
#### System information:
- Debian GNU/Linux 10.3 (Buster).
- DBeaver 6.3.5.202002151353.

#### Connection specification:
- PostgreSQL 11 (postgresql-11.7-0+deb10u1).
- PostgreSQL JDBC Driver 42.2.5.
- SSH tunnel.

#### Describe the problem you're observing:

When I try to import a CSV file that has less columns than the destination table, the assistant throws an exception like this:

```
Can't load entity attributes

Reason:
Index 3 out of bounds for length 3
```

This exception continues popping-up everytime that I re-map a column of the CSV with a column of the table.

#### Steps to reproduce, if exist:

. Create a table with 5 columns.
. In the tables tree, click on the table and choose `Import Data` → `CSV`.
. Import a CSV file with 3 columns.

#### Include any warning/errors/backtraces from the logs

There is no stack trace.

**Comment by patrickmdnet** [comment]
possible dup #7725

**Comment by uslss** [comment]
seems that the issue is already fixed in the scope of #7725.
you can test it in EA version https://dbeaver.io/files/ea/
this ticket will be closed. If the issue still reproducible, feel free to reopen the ticket

**Comment by randym127** [comment]
I still get this error with latest version, I have downgraded to 21.1.5 in the hopes that it fix it.
